src/applications/coronavirus-vaccination-expansion/tests/covid-vaccine.cypress.spec.js

```diff
@@ -11,27 +11,106 @@ const testConfig = createTestConfig(
     },
 
     pageHooks: {
-      introduction: ({ afterHook }) => {
-        afterHook(() => {
-          cy.findAllByText(/start/i, { selector: 'button' })
-            .last()
-            .click();
-        });
+      introduction: () => {
+        // cy.injectAxe();
+        cy.get('#introductionRadios-1').check();
+        cy.findByText(/continue/i, { selector: 'button' })
+          .first()
+          .click();
+
+        cy.injectAxe();
+      },
+      verify: () => {
+        cy.fillPage();
+      },
+      'vaccine-location': () => {
+        cy.wait('@getFacilities');
+        cy.get('.errorable-radio-button > input')
+          .first()
+          .check();
+        // cy.get('#location-0').check();
+      },
+      confirmation: () => {
+        cy.log('TODO - Tests for confirmation screen here');
       },
     },
 
     setupPerTest: () => {
-      // Log in if the form requires an authenticated session.
-      // cy.login();
+      cy.intercept('GET', '/covid_vaccine/v0/facilities/97214', {
+        statusCode: 200,
+        body: {
+          data: [
+            {
+              id: 'vha_648',
+              type: 'vaccination_facility',
+              attributes: {
+                name: 'Portland VA Medical Center',
+                distance: 2.31,
+                city: 'Portland',
+                state: 'OR',
+              },
+            },
+            {
+              id: 'vha_663',
+              type: 'vaccination_facility',
+              attributes: {
+                name: 'Seattle VA Medical Center',
+                distance: 142.35,
+                city: 'Seattle',
+                state: 'WA',
+              },
+            },
+            {
+              id: 'vha_653',
+              type: 'vaccination_facility',
+              attributes: {
+                name: 'Roseburg VA Medical Center',
+                distance: 161.95,
+                city: 'Roseburg',
+                state: 'OR',
+              },
+            },
+            {
+              id: 'vha_687',
+              type: 'vaccination_facility',
+              attributes: {
+                name: 'Jonathan M. Wainwright Memorial VA Medical Center',
+                distance: 209.89,
+                city: 'Walla Walla',
+                state: 'WA',
+              },
+            },
+            {
+              id: 'vha_692',
+              type: 'vaccination_facility',
+              attributes: {
+                name: 'White City VA Medical Center',
+                distance: 212.66,
+                city: 'White City',
+                state: 'OR',
+              },
+            },
+          ],
+        },
+      }).as('getFacilities');
 
-      cy.route('POST', formConfig.submitUrl, { status: 200 });
+      cy.intercept('POST', '/covid_vaccine/v0/expanded_registration', {
+        statusCode: 200,
+        body: {},
+      }).as('submitForm');
     },
 
     // Skip tests in CI until the form is released.
```

Comment:
is this page an application or a widget? if it's an application it shouldn't need a content page

Reply:
That is the API endpoint the form submits to

Reply:
oh i was referring to lines 116-118:
```js
    // Skip tests in CI until the form is released.
    // Remove this setting when the form has a content page in production.
    skip: Cypress.env('CI'),
```

is this test running correctly in Jenkins?